### PR TITLE
Outsource initialization of Bootstrap popovers (Fix "New comment" button)

### PR DESCRIPTION
**Pull request by Frodo161**
This branch fixes issue #524: Open the show_comment page. Hit the "new comment" button, then click "abort". Hitting the "new comment" button again won't have any effect now.

The problem is that two variables in the edit javascript file are declared as constants while hitting the button twice causes them to be redefined.

Changing the variable declaration from "const" to "var" won't cause any problems in this situation and fixes the bug.

### CODE REVIEW

**Comment by Splines** [comment]
@Frodo161 You're change certainly gets the job done. However, the code you were touching is actually duplicated code. Bootstrap [requires](https://getbootstrap.com/docs/5.3/components/popovers/#enable-popovers) to manually initialize popovers. When we dynamically add content to the page, we have to reinitialize this. I outsourced this behavior to a new function in `app/assets/javascripts/bootstrap_popovers.js` which is called from the respective places, so that we don't have the same code all over the place.

If you are fine with this, please make sure to test that popovers work everywhere, e.g. test popovers at some places, but especially in places where content is loaded dynamically, e.g. when you add a new chapter and a modal opens for that operation that has some popovers (helptexts) in it.

File: app/views/quizzes/proceed.coffee [hunk 1]
```diff
@@ -79,7 +79,6 @@ displayNext = ->
   <% else %>
   renderNext('<%= j render partial: "quizzes/quiz_round",
                            locals: { hidden: true } %>')
-  $('[data-bs-toggle="popover"]').popover()
```
Comment (Frodo161) [inline app/views/quizzes/proceed.coffee:82]: I'm not shure, why you didn't insert an "initBootstrapPopovers()" here.
I tried a quiz with and without this additional line, but I didn't see that it changed anything in the behavior. So probably, this is fine as it is, but maybe double check this.

File: app/assets/javascripts/upload.coffee [hunk 2]
```diff
@@ -439,8 +438,6 @@ directUpload provides an interface to upload (multiple) files to an endpoint
 ###
 @result = undefined
 @userManuscriptUpload = (fileInput) ->
-  # update helpdesk
-  $('[data-bs-toggle="popover"]').popover()
```
Comment (Frodo161) [inline app/assets/javascripts/upload.coffee:443]: Why didn't you add an "initBootstrapPopovers()" here?

File: app/views/quizzes/proceed.coffee (hunk 1 above)
Comment (Splines) [reply app/views/quizzes/proceed.coffee:82]: forgot to add it here, fixed it (even though you didn't notice a change when deleting the line, but it doesn't harm to have it here as well)

File: app/assets/javascripts/upload.coffee (hunk 2 above)
Comment (Splines) [reply app/assets/javascripts/upload.coffee:443]: Oh, my bad, I forgot it, fixed it.

**Review by Frodo161 — COMMENTED** [review]
I didn't see anything in the code (except the two missing lines you already added).

In my local MaMpf environment I tested some popovers (chapters, sections, lessons, comments and some more) and everything worked.

**Review by Splines — APPROVED** [review]

**Pull request merged**